### PR TITLE
DM-3514: Update header logo for WGB compliance

### PR DESCRIPTION
**Pull request by camillevilla**
### JIRA issue link
https://agile6.atlassian.net/browse/DM-3514

## Description - what does this code do?
- This code removes the VA logo from the header, in compliance with WGB. See #538 / DM-3459 for related WGB compliance work in footer.

## Testing done - how did you test it/steps on how can another person can test it 
1. Open website and look at header. There should be no VA seal. 
2. Resize the page to tablet and mobile widths. The proportion of "VA" to "Diffusion Marketplace" should change slightly in the mobile width.

### Before
Screen Shot 2022-08-16 at 8 56 45 AM
Screen Shot 2022-08-16 at 8 56 37 AM

### After
Screen Shot 2022-08-16 at 8 56 23 AM
Screen Shot 2022-08-16 at 8 56 17 AM

## Link to mock-ups/mock ups (image file if you have it) (if applicable)
https://www.figma.com/file/dkvSpADVqwN5KKByF1TVIa/Design-System---VA-DM?node-id=1305%3A1714

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Unit tests written (if applicable)
- [x] e2e/accessibility tests written (if applicable)
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating JIRA issue
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by camillevilla** [comment]
✅ Got the confirmation from design. I'll wait until #538 is in `master` before I rebase, squash and merge.

**Review by JoshingYou1 — COMMENTED** [review]
Hey Camille, can you point to `dm-3459` so I can look at only your changes? With it set to master, I see all of my changes as well, and it's a lot to sift through hehe

**Review by bradjohnson92008 — APPROVED** [review]
Looks good - great work Camille!

**Review by JoshingYou1 — APPROVED** [review]

**Pull request merged**
